### PR TITLE
Update build artifacts

### PR DESCRIPTION
**Pull request by stevemessick**
Drop 3.0.1; update 3.1; add 3.2.

@devoncarew

### CODE REVIEW

File: resources/META-INF/studio-contribs.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
   </extensions>
 
   <extensions defaultExtensionNs="com.android.project">
-    
+    <projectsystem implementation="io.flutter.project.FlutterProjectSystemProvider"/>
```
Comment (devoncarew) [inline resources/META-INF/studio-contribs.xml]: Is this change intended?

Comment (stevemessick) [reply resources/META-INF/studio-contribs.xml]: Yes.
